CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.6.1 / 2017-04-28
+
+- [BUGFIX] Fix broken OpsGenie integration
```

Comment:
Yes I realize that, just wasn't sure how to phrase it to be meaningful from a user's perspective. Wdyt of `Fix incorrectly handled hash for notification providers`?